### PR TITLE
Adjust DNS records to include terminal period

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1770960

ref: https://github.com/openshift/openshift-docs/issues/18027

### CODE REVIEW

**Comment by jboxman** [comment]
Preview: http://file.bos.redhat.com/jboxman/20200319/bz-1770960/installing/installing_vsphere/installing-vsphere.html#installation-dns-user-infra_installing-vsphere

File: modules/installation-dns-user-infra.adoc [hunk 1]
```diff
@@ -67,9 +67,9 @@ following records:
 [options="nowrap"]
 ----
 # _service._proto.name.                            TTL    class SRV priority weight port target.
-_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-0.<cluster_name>.<base_domain>.
-_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-1.<cluster_name>.<base_domain>.
-_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-2.<cluster_name>.<base_domain>.
+_etcd-server-ssl._tcp.<cluster_name>.<base_domain>.  86400 IN    SRV 0        10     2380 etcd-0.<cluster_name>.<base_domain>.
```
Comment (zhaozhanqi) [inline modules/installation-dns-user-infra.adoc]: I do not think we need the `.` at the end of `etcd-0.<cluster_name>.<base_domain>.`
@lihongan ^^

Comment (jboxman) [reply modules/installation-dns-user-infra.adoc]: @zhaozhanqi, does vSphere have some kind of DNS service? Where are you creating the DNS record?

I want to be sure that the issue isn't one of different DNS services expecting different input; Maybe one requires the terminal period, and another rejects it or accepts input both with and without the terminal period.

Comment (zhaozhanqi) [reply modules/installation-dns-user-infra.adoc]: we are using AWS route53 for vsphere cluster testing @jiajliu ^^ could you help confirm if I'm correct for above DNS records.  thanks

File: modules/installation-dns-user-infra.adoc [hunk 2]
```diff
@@ -43,22 +43,22 @@ API calls can fail, and you cannot retrieve logs from Pods.
 ====
 
 |Routes
-|`*.apps.<cluster_name>.<base_domain>`
+|`*.apps.<cluster_name>.<base_domain>.`
```
Comment (lihongan) [inline modules/installation-dns-user-infra.adoc]: The update is correct, the `.` at the end of domain is required.

Comment (zhaozhanqi) [reply modules/installation-dns-user-infra.adoc]: yes, the name should have `.` at the end. but seems the `target` part of etcd SRV record `etcd-0.<cluster_name>.<base_domain>.` should not contain `.` at the end.
`_etcd-server-ssl._tcp.<cluster_name>.<base_domain>.  86400 IN    SRV 0        10     2380 etcd-0.<cluster_name>.<base_domain>.`

Comment (lihongan) [reply modules/installation-dns-user-infra.adoc]: yes, seems not required for target part.

**Comment by jboxman** [comment]
@zhaozhanqi @lihongan 

Updated.

**Comment by mburke5678** [comment]
@jboxman LGTM

**Pull request merged**